lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -294,3 +297,95 @@ twoPartyBalanceHandler p1 p2 currency = do
     (ServerState pool configTVar _) <- ask
     ucacAddresses <- fmap lndrUcacAddrs . liftIO $ readTVarIO configTVar
     liftIO . withResource pool $ Db.twoPartyBalance p1 p2 (getUcac ucacAddresses currency)
+
+
+multiSettlementHandler :: [CreditRecord] -> LndrHandler NoContent
+multiSettlementHandler transactions = do
+    result <- mapM (uncurry submitMultiSettlement) $ zip [0..(length transactions)] transactions
+    return $ head result
+
+
+submitMultiSettlement :: Int -> CreditRecord -> LndrHandler NoContent
+submitMultiSettlement recordNum signedRecord@(CreditRecord creditor debtor _ memo submitterAddress _ hash sig _ _ _ _) = do
```

Comment:
Now that the design of this change has been settled, DRY this code with the `submitHandler` code above.